src/components/Board.js

```diff
@@ -8,26 +8,117 @@ import NewCardForm from './NewCardForm';
 import CARD_DATA from '../data/card-data.json';
 
 class Board extends Component {
-  constructor() {
-    super();
+
+  constructor(props) {
+    super(props);
 
     this.state = {
       cards: [],
     };
   }
 
+  componentDidMount() {
+
+    const url = `${this.props.url}/boards/katrina/cards`;
```

Comment:
I like how you constructed this URL! Maybe you could've used the prop `boardName` if you had set it to `katrina` (passed in from `App`)